controls/raddocking/features/document-host.md

```diff
@@ -123,6 +123,28 @@ __Figure 2:__ __DocumentHost__ with visible __HorizontalScrollBar__ and drop dow
 
 ![](images/raddocking-features-document-host-6.png)
 
+## Removing Default Padding
+
+To remove the extra space around the panes in the pane groups and the `DocumentHost`, you need to create an appropriate style for the `RadPaneGroup` element and override the default `Padding`.

```

Comment:
"..you need to create.." - generally speaking we avoid phrases like "need to", "must", "should". They are not forbidden (maybe except "must" and sometimes "should"), but preferably, we use "you can", which gives the reader more freedom. In this case, I think "..you can create..." is more suitable. My personal rule here is that if there are more than one options to achieve the requirement, use "you can", otherwise, use "you will need to".